src/main/java/com/exadel/sandbox/dto/TgInfoDto.java

```diff
@@ -0,0 +1,22 @@
+package com.exadel.sandbox.dto;
```

Comment:
About the package, let's use feature-oriented structure. EX: do not create DTO package and put all DTOs in this package, but create package employee and put all information about him to this package